src/UnityMvvmToolkit.UnityPackage/Assets/Plugins/UnityMvvmToolkit/Runtime/UITK/BindableUIElements/Uxmls/BindableDropdownField.Uxml.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine.UIElements;
+using UnityMvvmToolkit.UITK.Extensions;
+
+namespace UnityMvvmToolkit.UITK.BindableUIElements
+{
+    public partial class BindableDropdownField
```

Comment:
It is better not to set `public` or another modifier out of the main partial class. Because if you need to change the class modifier in the future, you will change it only in one place.